### ISSUE TITLE
Create a timeline

### ISSUE THREAD

**Issue by RocioEspeche** [RocioEspeche/homepage#8]
Add a list of all your activities, achievements, work experience. Add as much info as you can remember. In case you don’t have working experience tell about your pet-projects or student projects. Don’t be shy :)

**Comment by RocioEspeche** [comment]
The first project I worked on was the e-commerce website for a fictional book store. It was during a frontend development course. Initially, we were 3 members in the group. Later, one member dropped out. That's where I learned a lot about HTML and CSS. Later on, we also learned DOM. Afterward, I started the full stack development course at Digital House, and there we created a page called "Mercado Liebre," which was a replica of Mercado Libre's homepage. This project was individual, and we carried it out using HTML and CSS. Finally, the final project to pass this course was an e-commerce website for a bicycle shop in Santiago del Estero, Argentina, owned by one of my group mates. We were 6 in total. There, we created wireframes in Figma and then used HTML, CSS, JavaScript, Node.js, Express, MySQL, and React to build the website. Additionally, we applied agile methodologies (Scrum), so we used Trello board to organize tasks. I feel like I learned a lot and I'm ready for new challenges. Always seeking more.
